src/components/Avatar/styles.scss

```diff
@@ -28,7 +28,7 @@ $avatar-status-indicator-size-large: 14px;
 		height: 100%;
 
 		color: transparent;
-		border-radius: $default-border-radius;
+		border-radius: 50%;
```

Comment:
Do we want to change this for the current UI as well?

Reply:
Yes, As @Shailesh351 mentioned this morning, we want to make Avatars round